pages/common/encfs.md

```diff
@@ -0,0 +1,24 @@
+# encfs
+
+> Encrypted virtual filesystem.
+> More information: <https://github.com/vgough/encfs>.
+
+- Initialize or mount an encrypted filesystem:
+
+`encfs {{/path/to/cipher_dir}} {{/path/to/mount_point}}`
+
+- Initialize an encrypted filesystem with standard settings:
+
+`encfs --standard {{/path/to/cipher_dir}} {{/path/to/mount_point}}`
+
+- Run encfs in foreground instead of spawning a daemon:
+
+`encfs -f {{/path/to/cipher_dir}} {{/path/to/mount_point}}`
+
+- Unmount an encrypted filesystem:
+
+`fusermount -u {{/path/to/mount_point}}`
```

Comment:
Why `fusermount`?  Do you mean this?
```suggestion
`encfs -u {{/path/to/mount_point}}`
```

Reply:
Using `fusermount` is the recommended way in the [`encfs` manpage](https://linux.die.net/man/1/encfs), even though `encfs -u` indeed exists. Should I stick to using the same program even though it is not really the recommended way to use it?

Reply:
Hmm, I'm really not sure.  @sbrl?

Reply:
I this this should be in the header:
```
> For unmount filesystems use `fusermount`.
```
Something like that. But I think we don´t include examples for another commands (maybe there is 1 exception around).

Reply:
@navarroaxel applied it at https://github.com/tldr-pages/tldr/pull/5513/commits/5b3f78819def3bfb54af04c318ca1035eed13f78

Reply:
:thinking: I think we should add the `fusermount` page. maybe we can merge it before this.